### PR TITLE
Chore/mock auth

### PR DESCRIPTION
**Pull request by sidhantas**
Problem
=======
C2D-44

Solution
========
What I/we did to solve this problem
*  Created a manual mock of the useAuth0 method from auth0-react

Change Summary:
---------------
* Created  useAuth0 mock

Images/Important Notes (optional):
-----------------------
* If anyone has any requests for other mocks please let me know

### CODE REVIEW

File: __mocks__/@auth0/auth0-react.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+const useAuth0 = () => ({
+  user: "random user",
```
Comment (krishnakpandian) [inline __mocks__/@auth0/auth0-react.js]: Can we update this `user` object to also include fields for
`email_verified`
`email`
`https://cruzhacks.com/roles`
`sub`

Comment (krishnakpandian) [reply __mocks__/@auth0/auth0-react.js]: Along with this, have it so email_verified can either be tested with true or false. https://cruzhacks.com/roles will have an array of values either including the value "Organizer" or "User"

**Review by krishnakpandian — CHANGES_REQUESTED** [review]

File: __mocks__/@auth0/auth0-react.js (hunk 1 above)
Comment (sidhantas) [reply __mocks__/@auth0/auth0-react.js]: Sorry, I'm kind of confused for the third field, from what I'm reading the object should look like this:
`{
  email_verified: true/false,
  email: "mock email",
  roles: ["Organizer", "User"],
  sub: "mock user"
}`

Correct me if I'm wrong here

Comment (krishnakpandian) [reply __mocks__/@auth0/auth0-react.js]: What you have is correct, but roles will only contain Organizer or User. Auth0 stores roles as an array within the idToken.

Comment (krishnakpandian) [reply __mocks__/@auth0/auth0-react.js]: @sidhantas One other thing to correct is that the title of the field is "https://cruzhacks.com/roles"

Comment (sidhantas) [reply __mocks__/@auth0/auth0-react.js]: Ah, I got it, just had to put the token into jwt.io to figure it out

**Review by krishnakpandian — APPROVED** [review]

**Review by w-gao — APPROVED** [review]
This looks like it should work. Thanks @sidhantas!

**Pull request merged**